src/sentry/api/endpoints/organization_teams.py

```diff
@@ -169,29 +187,41 @@ def post(self, request: Request, organization, **kwargs) -> Response:
                     },
                     status=409,
                 )
-            else:
-                team_created.send_robust(
-                    organization=organization, user=request.user, team=team, sender=self.__class__
-                )
             if self.should_add_creator_to_team(request):
                 try:
                     member = OrganizationMember.objects.get(
```

Comment:
nit: This read could be done before the writes. That way if this read fails, then we don't have to write anything.

Reply:
Ok I added it right after the team is created inside the atomic block